bedrock/firefox/templates/firefox/includes/download-button.html

```diff
@@ -64,6 +64,9 @@
               {% if channel != 'alpha' %}
                 <span class="download-subtitle">{{ _('Get it free on Google Play') }}</span>
               {% endif %}
+            {% elif plat.os == 'ios' %}
```

Comment:
Let's add <code>waffle.switch('ios-active')</code> (after the platform check) so there's no risk of this being exposed in production until iOS actually launches.